src/Admin/AbstractAdmin.php

```diff
@@ -514,42 +514,30 @@ abstract class AbstractAdmin implements AdminInterface, DomainObjectInterface, A
     /**
      * The form group disposition.
      *
-     * NEXT_MAJOR: must have `[]` as default value and remove the possibility to
-     * hold boolean values.
-     *
-     * @var array|bool
+     * @var array<string, mixed>
      */
-    private $formGroups = false;
+    private $formGroups = [];
```

Comment:
Will this work? AFAIK the bool value is used to indicate a not initialized group.

Reply:
@phansys already deprecated the `false` value. I just removed it.

Currently the $formGroups seems to be used like it's always an array.
We're just lucky that
```
if (!isset($groups[$code])) {
```
works when $group is `false`;

Reply:
Is it ok for you @core23 ?